### PR TITLE
Fix/rubocop style mitigations

### PR DESCRIPTION
**Pull request by nysthee**
First part of a bunch of ruby-style-guide mitigations.

### CODE REVIEW

File: Guardfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 rspec_options = {
   version:        1,
   all_after_pass: false,
-  all_on_start:   false,
```
Comment (jnunemaker) [inline Guardfile]: I actually prefer the trailing comma. Having it means never needing to remember it when you add a new entry and it also keeps it from showing up in the git history.

Comment (nysthee) [inline Guardfile]: I can revert the commit if you want.

**Comment by jnunemaker** [comment]
Commented on all the style changes I'm not interested in. The rest look great! Thanks for doing it. If you would swap the ones I mentioned back I would be happy to merge.

**Comment by nysthee** [comment]
I wil revert the commented commits.
Also I found this weird sentence; "When you HTTParty, you must party hard!" Is this meant to be?

**Comment by nysthee** [comment]
:)

**Comment by jnunemaker** [comment]
> Also I found this weird sentence; "When you HTTParty, you must party hard!" Is this meant to be?

Weird...we should remove it. HAHAHAHHAAHHAHAHAHA

**Comment by nysthee** [comment]
Should be ok now.

File: Guardfile (hunk 1 above)
Comment (jnunemaker) [inline Guardfile]: :+1:

**Comment by jnunemaker** [comment]
Did you mean to close this?

**Comment by nysthee** [comment]
Things got a bit messy and I'm redoing it.

**Comment by jnunemaker** [comment]
Ok!

**Pull request closed without merging**